app/containers/App/App.js

```diff
@@ -3,19 +3,16 @@ import React from 'react';
 import TeamMembers from '../../containers/TeamMembers';
 import GamesList from '../../containers/GamesList';
 
-class App extends React.Component {
-
-  render() {
-    return (
-      <div>
-        <div style={leftColumnStyle}>
-          <GamesList />
-        </div>
-        <div style={rightColumnStyle}>
-          <TeamMembers />
-        </div>
-      </div>);
-  }
+function App() {
```

Comment:
You've been listening to Tyler too much! Arrow functions all the way!

`const App = () = > (<div>...</div>);`

In all seriousness though - this is completely fine too